modules/3d/include/opencv2/3d/volume_settings.hpp

```diff
@@ -0,0 +1,213 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html
+
+
+#ifndef OPENCV_3D_VOLUME_SETTINGS_HPP
+#define OPENCV_3D_VOLUME_SETTINGS_HPP
+
+#include <opencv2/core/cvstd.hpp>
```

Comment:
`core.hpp` should be used here.
Direct files should be used in cases when they are not included by `core.hpp` by default.

Reply:
done